src/mentionediting.js

```diff
@@ -98,6 +101,28 @@ export function _toMentionAttribute( viewElementOrMention, data ) {
 	return _addMentionAttributes( baseMentionData, data );
 }
 
+// Converter that prevents other converters from converting partial mentions.
+//
+// This helper consumes partial mention which should not be downcasted to allow usage
+// of attributeToElement() converter to override default conversion without the need to write more specialised converter.
```

Comment:
I read the code and it was clearer than this comment 😜 This one is mind-boggling.